libraries/AP_BattMonitor/LogStructure.h

```diff
@@ -55,8 +56,41 @@ struct PACKED log_BCL {
     uint16_t cell_voltages[12];
 };
 
+// @LoggerMessage: BATI
+// @Vehicles: All
+// @Description: Smart Battery Information
+// @Field: TimeUS: Time since system startup
+// @Field: id: battery monitor instance number
+// @Field: func: battery function
+// @Field: type: battery type (chemistry)
+// @Field: nSer: number of battery cells in series
+// @Field: capD: design capacity when full according to manufacturer, -1: field not provided.
+// @Field: capF: capacity when full (accounting for battery degradation), -1: field not provided.
+// @Field: Vmax: maximum charging voltage, -1: field not provided.
+// @Field: cyc: charge/discharge cycle count. UINT16_MAX: field not provided.
+// @Field: ser: serial number in ASCII characters, 0 terminated. All 0: field not provided.
+// @Field: date: manufacture date (DD/MM/YYYY) in ASCII characters, 0 terminated. All 0: field not provided
+// @Field: name: product name encoded as 'device name'_'manufacturer'
+struct PACKED log_BATI {
```

Comment:
should be a log write() call

Reply:
need to test that the log of the strings works

Reply:
switched to log_write and tested. 
![image](https://user-images.githubusercontent.com/69225461/121989943-f2488b00-cd6a-11eb-9efb-3bd711545861.png)